src/com/lftechnology/java_training/jan12/WordManipulator.java

```diff
@@ -0,0 +1,88 @@
+package com.lftechnology.java_training.jan12;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Takes the words in its argument list and prints out any duplicate words, the number of distinct words, and a list of the words with
+ * duplicates eliminated. Hint: Try with HashSet and List implementation.
+ * 
+ * @author Utsab Neupane<utsabneupane@lftechnology.com>
+ * 
+ */
+public class WordManipulator {
+	private static final String[] WORDS_LIST = { "hello", "world", "how", "are", "you", "hello", "what", "are", "you", "doing" };
+	private static final Logger LOGGER = Logger.getLogger(WordManipulator.class.getName());
+
+	public static void main(String[] args) {
+		List<String> listOfWords = new ArrayList<>();
+		// store all words in a list from array of words
+		for (String words : WORDS_LIST) {
+			listOfWords.add(words);
+		}
+		LOGGER.log(Level.INFO, "list of words: {0}", listOfWords);
+		Set<String> dupEliminatedWords = new HashSet<>();
+		Set<String> dupWords = new HashSet<>();
+		Set<String> distinctWords = new HashSet<>();
+		// get duplicate eliminated words
+		dupEliminatedWords = getDupEliminatedWords();
+		// get duplicate words
+		dupWords = getDupWords();
+		// get distinctWords
+		distinctWords = getDistinctWords();
+		LOGGER.log(Level.INFO, "duplicate eliminated words: {0}", dupEliminatedWords);
+		LOGGER.log(Level.INFO, "duplicate words:{0}", dupWords);
+		LOGGER.log(Level.INFO, "no of distinct words:{0} \n distinct words:{1}", new Object[] { distinctWords.size(), distinctWords });
+	}
+
+	/**
+	 * This method eliminate duplicate words and return {@link Set} of words without duplicate
+	 * 
+	 * @return {@link Set} that contains duplicate eliminated words
+	 * @author Utsab Neupane<utsabneupane@lftechnology.com>
+	 */
+	private static Set<String> getDupEliminatedWords() {
```

Comment:
Isn't a single line , `Set<String> dupEliminatedWords = new HashSet<>(listOfWords);` suffice for what this method does.

You are using Set here so, If you intialzie a set with duplicate values, it will automcatically eleminates those and give you a distinct set of result.